src/upstream_rbus.c

```diff
@@ -241,3 +244,87 @@ void eventReceiveHandler( rbusHandle_t rbus_Handle, rbusEvent_t const* event, rb
     }
 }
 #endif
+
+rbusError_t SendRbusEventCloudConnOnline()
+{
+	rbusError_t rc = RBUS_ERROR_BUS_ERROR;
+	if(cloud_online_subscribe)
+	{
+		rbusEvent_t event = {0};
+		rbusObject_t data;
+		rbusValue_t value;
+
+		ParodusInfo("publishing cloud connection online Event\n");
+
+		rbusValue_Init(&value);
+		rbusValue_SetInt32(value, 1);
+
+		rbusObject_Init(&data, NULL);
+		rbusObject_SetValue(data, "value", value);
+
+		event.name = CLOUD_CONN_ONLINE;
+		event.data = data;
+		event.type = RBUS_EVENT_GENERAL;
+
+		rc = rbusEvent_Publish(rbus_Handle, &event);
+
+		rbusValue_Release(value);
+		rbusObject_Release(data);
+
+		if(rc != RBUS_ERROR_SUCCESS)
+			ParodusError("provider: rbusEvent_Publish cloud connection online event failed: %d\n", rc);
+	}
+	return rc;
+}
+
+
+rbusError_t CloudConnSubscribeHandler(rbusHandle_t handle, rbusEventSubAction_t action, const char* eventName, rbusFilter_t filter, int32_t interval, bool* autoPublish)
+{
+    (void)handle;
+    (void)filter;
+    (void)autoPublish;
+    (void)interval;
+
+    ParodusInfo(
```

Comment:
Remove the \n newlines in this log and add log similar to this https://github.com/xmidt-org/webcfg/blob/master/src/webcfg_rbus.c#L1057